keadm/cmd/keadm/app/cmd/util/set.go

```diff
@@ -0,0 +1,338 @@
+/*
+Copyright The Helm Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package strvals
+
+import (
+	"fmt"
+	"reflect"
+	"strconv"
+	"strings"
+	"unicode"
+)
+
+// ParseSetByCommma splits a set line according to the comma.
+//
+// A set line is of the form name1=value1,name2=value2,...,nameM=valueM
+func ParseSetByComma(set string) []string {
```

Comment:
You can define some functions that are not external as private functions, and comments need to be modified synchronously.